src/scanner/images/index.ts

```diff
@@ -75,10 +88,11 @@ export function getImageParts(imageWithTag: string) : {imageName: string, imageT
 export async function scanImages(images: IPullableImage[]): Promise<IScanResult[]> {
   const scannedImages: IScanResult[] = [];
 
-  for (const { imageName, fileSystemPath, imageWithDigest } of images) {
+  for (const { imageName, fileSystemPath, imageWithDigest, skopeoRepoType } of images) {
     try {
       const shouldIncludeAppVulns = true;
-      const dockerArchivePath = `docker-archive:${fileSystemPath}`;
+       const archiveType= skopeoRepoType == SkopeoRepositoryType.DockerArchive?"docker-archive":"oci-archive";
```

Comment:
I'm curious how this passed our linter 🤔  We definitely need to fix our linter and maybe introduce `prettier` in a separate PR. 
This line will require only adding some spaces between `=`, `?` and `:`.
@chen-keinan it's great that you were able to help to catch this :)

Reply:
It doesn't need spaces but I didn't comment on style in this PR because it's pointless, we can fix all of this by introducing `prettier` to format the code rather than asking people to remember